### ISSUE TITLE
dynamic field - padding / background css conflict

### ISSUE THREAD

**Issue by rtpHarry** [CrocoBlock/suggestions#2840]
**Describe the bug**
Either the padding or the background is not being applied to the correct div with a dynamic field.

If you set a background colour on a dynamic field, and then add padding, the padding is on a higher level div so it works more like a margin:

![image](https://user-images.githubusercontent.com/1038062/111806194-a439fd80-88c9-11eb-9c8a-de06aa24e43a.png)

![image](https://user-images.githubusercontent.com/1038062/111806236-b1ef8300-88c9-11eb-9a64-7af3b4c2e46e.png)

![image](https://user-images.githubusercontent.com/1038062/111806347-caf83400-88c9-11eb-8639-87f46be94634.png)

(I'm using absolute positioning in the example above but I tested disabling and it didn't affect it)

**To Reproduce**

1. Add a dynamic field
2. Set a background colour (probably all bg types)
3. Set the padding

**Expected behavior**
Adding the padding should pad the inside area of the element, but it adds it to the outside of the background div like a margin.

**Screenshots**
Above.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: MacBook Pro
 - OS: macOS Big Sur
 - Browser Chrome
 - Version Latest

**Additional context**

**Comment by rtpHarry** [comment]
**Update:**

Actually this is not exactly what I thought it was.

It is a second set of paddings on the Advanced tab which is causing the bug reported above. I set the position inline then just added the padding on the settings at the top of that tab.

I've just now found there is an additional set of margin/paddings on the Style tab.

Using the padding within the Style tab works as expected, but using the Advanced tab padding is broken.
